src/mesh/Mesh.hpp

```diff
@@ -113,6 +113,7 @@ namespace polyfem
 
 		//Nodal access
 		virtual RowVectorNd point(const int global_index) const = 0;
+		virtual int cell_vertex_(const int f_id, const int lv_id) const = 0;
```

Comment:
why `_` in `cell_vertex_`?

Reply:
I put a `_` at the end because there's a function which is not virtual and has the same name in `Mesh3D` class. Otherwise can I change the original `cell_vertex` in `Mesh3D` to virtual? (Sorry this is my first time using a virtual function)

Reply:
so remove the _ from `Mesh.hpp` and make `cell_vertex` in `Mesh3d` `override`, no need to make them virtual